app/src/main/java/com/kickstarter/features/pledgedprojectsoverview/ui/PledgedProjectsOverviewScreen.kt

```diff
@@ -129,6 +133,7 @@ fun PledgedProjectsOverviewScreen(
                         PPOCardView(
                             viewType = it.viewType,
                             onCardClick = { },
+                            onProjectPledgeSummaryClick = { onProjectPledgeSummaryClick(it.backingDetailsUrl) },
```

Comment:
@leighdouglas Here's where we pass the backingDetailsUrl from the PPO card when user clicks on the tap area.